codejail/tests/test_safe_exec.py

```diff
@@ -67,7 +71,7 @@ def test_printing_stuff_when_you_shouldnt(self):
     def test_importing_lots_of_crap(self):
         globs = {}
         self.safe_exec(textwrap.dedent("""\
-            from numpy import *
+            from os import *
```

Comment:
I don't remember the goal of this test, but it may have been to stress the memory or time limits to be sure we had them set reasonably for the kinds of big packages the real jails would be dealing with.  I'm not sure replacing numpy with os is a good idea.